### PR TITLE
Upgrade Unity to 2021.3.5f1

### PR DESCRIPTION
**Pull request by moreal**
This pull request upgrades the Unity Editor version to 2021.3.5f1, the Unity Editor version of the current NineChronicles.

You can see the build result in https://github.com/moreal/xunit-unity-runner/actions/runs/2779124711.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 xUnit.net Unity Runner Changelog
 ================================
 
+
+Version 0.6.0
+-------------
+
+To be released.
+
+ -  Upgraded Unity Editor from 2020.3.4f1 to 2021.3.5f1.
```
Comment (dahlia) [inline CHANGES.md]:
```suggestion
 -  Upgraded Unity from 2020.3.4f1 to 2021.3.5f1.
```

File: Assets/Resources/BillingMode.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```
Comment (dahlia) [inline Assets/Resources/BillingMode.json]: I guess this file is unnecessary.

**Review by dahlia — COMMENTED** [review]
Thank you for upgrading!

File: Assets/Resources/BillingMode.json (hunk 2 above)
Comment (moreal) [reply Assets/Resources/BillingMode.json]: I agree. Maybe the `Resources/*` files are unnecessary 🤔 I'll remove them and add the path to `.gitignore`

**Review by dahlia — APPROVED** [review]

**Review by dahlia — CHANGES_REQUESTED** [review]
The build is apparently failing.  Could you take a look into this?

**Comment by moreal** [comment]
> The build is apparently failing. Could you take a look into this?

When I looked, it was because I'm not authorized for this repository so it couldn't access the ULF (Unity License File) secret variable.

image

If you want to look at the build, you can see my forked repository's GitHub Actions result. Here is the link, https://github.com/moreal/xunit-unity-runner/actions/runs/2779124711 .

**Comment by dahlia** [comment]
Fair enough.  I'm going to merge this!

**Pull request merged**